view/templates/item/compose.tpl

```diff
@@ -94,3 +95,43 @@
 		</form>
 	</div>
 </div>
+<script>
+  Dropzone.autoDiscover = false;
+  var dropzoneCompose = new Dropzone( '#comment-edit-form-{{$id}}',  { 
+    paramName: "userfile", // The name that will be used to transfer the file
+    maxFilesize: 2, // MB
```

Comment:
This value should come from the internal size limit through a template variable. This size is given by `Strings::getBytesFromShorthand(DI::config()->get('system', 'maximagesize'))` but you have to divide by either 10^6 (MB) or 2^20 (Mb) to get the value for this library.

Reply:
Sorry... i tried something, but i have no idea, how to do this.
When i add declaration of variable in php, the composer returns blank white... 
I need help for this.

Reply:
I add to `src/Module/Item/Compose.php` the following line
```
                  $tpl = Renderer::getMarkupTemplate('item/compose.tpl');
                  return Renderer::replaceMacros($tpl, [
+                         '$max_imagesize'    => DI::config()->get('system', 'maximagesize'),
                          '$l10n' => [
....
```
and in `view/templates/item/compose.tpl` i added this debug-output:

```
  <script>
    Dropzone.autoDiscover = false;
+   var maxis = {{$max_imagesize}};
+   console.log("maxis", maxis);
    var dropzoneCompose = new Dropzone( '#comment-edit-form-{{$id}}',  { 
```

and then /compose gives me a white, blank page.

When i set a string in Compose.php, then it writes this string in the console... 

Is it possible, that maximagesize in system config is not set?

Reply:
Sorry for my partially functional example, you need to add `use Friendica\Util\Strings;` near the top of the `src/Module/Item/Compose.php` file for my snippet to work. Alternatively, you can use the following portable snippet: `\Friendica\Util\Strings::getBytesFromShorthand(DI::config()->get('system', 'maximagesize'))`.

Reply:
This is the error only with unmodified code, just on editing with DI:: set.
```
2023-03-13T16:14:43Z index [ERROR]: Uncaught Exception Error: "Class "Friendica\Module\Post\DI" not found" at /var/www/soc.schuerz.at/src/Module/Post/Edit.php line 186 {"exception":"Error: Class \"Friendica\\Module\\Post\\DI\" not found in /var/www/soc.schuerz.at/src/Module/Post/Edit.php:186\nStack trace:\n#0 /var/www/soc.schuerz.at/src/BaseModule.php(244): Friendica\\Module\\Post\\Edit->content()\n#1 /var/www/soc.schuerz.at/src/App.php(700): Friendica\\BaseModule->run()\n#2 /var/www/soc.schuerz.at/index.php(52): Friendica\\App->runFrontend()\n#3 {main}"} - {"file":null,"line":null,"function":null,"request-id":"640f4bf378653","uid":"49715c","process_id":1253003}
```

Reply:
ahhhh!!

adding `use Friendica\DI;` on top of the file solves the error. So i will fix the rest as expected now.